src/main/proto/netflix/titus/titus_base.proto

```diff
@@ -373,3 +374,19 @@ message Version {
   // incremented if some job or task entity is created/updated.
   // uint64 revision = 2;
 }
+
+// Definition of a request to add a platform sidecar alongside a task
+// Note that this is *not* a user-defined sidecar, that is why it just has a
+// name. These platform-sidecars are attached a task start time, and the
+// definition of what the sidecar is is not baked into the job itself, just the
+// intent.
+message PlatformSidecar {
```

Comment:
As this is also not a basic type, could you move it to its own file? I think moving it to `titus_containers.proto` would be also fine, as they are related to the same concept.

Reply:
relocated